src/main/java/me/mrCookieSlime/Slimefun/Objects/SlimefunItem/machines/electric/AutomatedCraftingChamber.java

```diff
@@ -68,6 +68,12 @@ public void newInstance(BlockMenu menu, Block b) {
 						return false;
 					});
 				}
+
+				menu.replaceExistingItem(7, new CustomItem(new ItemStack(Material.CRAFTING_TABLE), "&7Craft Last", "", "&e> Click to craft the last shaped recipe", "&cOnly works with the last one"));
+				menu.addMenuClickHandler(7, (p, slot, item, action) -> {
+					tickOnce(b);
```

Comment:
You could pretty much spam-click this button and craft a bunch of items very fast and without consuming power.